keyboards/40percentclub/luddite/keymaps/tzarc/config.h

```diff
@@ -0,0 +1,8 @@
+#include "../../config.h"
```

Comment:
The parent `config.h` is automatically included, so this line can be safely removed
```suggestion
```

Reply:
Removed.